Hunting Queries/OfficeActivity/OfficeMailForwarding_hunting.yaml

```diff
@@ -1,5 +1,5 @@
 id: 871ba14c-88ef-48aa-ad38-810f26760ca3
-name: Long lookback for multiple users forwarding Office mail to same destination
+name: UOffice Mail Forwarding
```

Comment:
Looks like you accidentally put a "U" at the beginning of the name

Reply:
Fixed. Thanks.